llvm/test/Analysis/DependenceAnalysis/PR51512.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: opt < %s -disable-output "-passes=print<da>" -aa-pipeline=basic-aa
+
+; Check that the testcase does not crash the compiler.
+; https://github.com/llvm/llvm-project/issues/51512
+
```

Comment:
Two questions about the test but maybe they are the same things:
- I see that this is a test that used to crash, but it is probably better to test the output of DA here?
- Can we check for FlagNSW?